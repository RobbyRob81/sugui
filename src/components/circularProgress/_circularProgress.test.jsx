import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import CircularProgress from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<CircularProgress />);
  expect(container.firstChild).toMatchSnapshot();
});

it('displays a text label', () => {
  const { getByText } = customRender(<CircularProgress label="Test" />);
  expect(getByText(/Test/i)).toBeVisible();
});
