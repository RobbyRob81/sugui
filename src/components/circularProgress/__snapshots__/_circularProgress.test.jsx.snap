// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  data-testid="progress"
>
  <div
    class="sc-bwzfXH epptpo"
    color="abbey"
  />
</div>
`;
